tools/revng-lift/BinaryFile.cpp

```diff
@@ -39,6 +39,12 @@ using LabelList = BinaryFile::LabelList;
 static Logger<> EhFrameLog("ehframe");
 static Logger<> LabelsLog("labels");
 
+// Hack-ish custom PT
+// TODO find a finer way
+constexpr int PT_PAGEBUSTER { 0x8 };
+
+static int g_epoch_counter { 0 };
```

Comment:
Do not use global variables, this can be a local variable just outside the loop.
Also make sure you follow the [LLVM Coding Conventions](https://llvm.org/docs/CodingStandards.html).

```suggestion
static int EpocCounter { 0 };
```